src/components/custom/_dhow-it-works.tsx

```diff
@@ -1,11 +1,14 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, { ElementType } from 'react'
 import styled from 'styled-components'
 import { localize } from 'components/localization'
 import { Container, SectionContainer } from 'components/containers'
 import { Header } from 'components/elements'
 import device from 'themes/device'
 
+type DHowItWorksProps = {
+    title: object
```

Comment:
We should avoid using `object` type as much as we can. Can you declare the object type by its keys and types pair?